cves/CVE-2024-10220.yaml

```diff
@@ -0,0 +1,26 @@
+cve: CVE-2024-10220
+issueUrl: https://github.com/kubernetes/kubernetes/issues/124531
```

Comment:
I think we should use https://github.com/kubernetes/kubernetes/issues/128885 instead